src/components/signup/SignUpButton.tsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+import { lightBlue } from '@mui/material/colors';
+import { Box, Button, CircularProgress } from '@mui/material';
+
+export default function CircularIntegration() {
```

Comment:
p5
```suggestion
const CircularIntegration = () => {
...
}

export default CircularIntegration;
```

혹시 요건 어떻게 생각하시나요?

Reply:
함수명까지 변경 했습니다
확인 부탁드릴게요!

Reply:
좋아유~😀